test/unit/sdk_reporter/test_sdk_reporter.py

```diff
@@ -104,16 +215,75 @@ def test_agent_configuration_when_configure_agent_throws_error(self):
             assert AgentConfiguration.get().should_profile is True
             assert AgentConfiguration.get().sampling_interval == timedelta(seconds=13)
 
-    def test_when_backends_sends_resource_not_found_it_stops_the_profiling(self):
+    def test_when_backends_sends_resource_not_found_it_stops_the_profiling_in_non_lambda_case(self):
         self.client_stubber.add_client_error('configure_agent', service_error_code='ResourceNotFoundException',
                                              service_message='Simulated error in configure_agent call')
+        os.environ.__setitem__(AWS_EXECUTION_ENV_KEY, 'AWS_Lambda_java')
```

Comment:
The test is about non-lambda, though this is set to lambda java. Either update the test name, either remove this line? (also related to my questions why are we considering there the java lambda usecase; I think it can't happen, and this should be removed).

Reply:
fixed this!